app/Http/Controllers/ArticleController.php

```diff
@@ -64,7 +64,7 @@ public function show($id): View|Factory|Application
         //show.blade.phpから渡されたidに該当するarticleを見つけ、詳細を表示する
         $article = Article::findOrFail($id);
 
-        return view('layouts/show')
+        return view('layouts/dash_detail')
```

Comment:
```suggestion
        return view('layouts/dashboard.detail')
```
他がdashboardなのに`dash_`になってるの不自然だから、統一した方がいい。
あと、多分viewファイルのnaming conventionはdot notationだからドット記法した方がいいと思う。

Reply:
hoge.detail.blade.phpのようにdotを3つ使うと上手く読み込めなかったため、dashboardDetail.blade.phpとしdotを2つ使ったところ読み込めた。よって後者をviewファイルの名前と致したい。